packages/larva-patterns/objects/o-collection-item/o-collection-item.floated.js

```diff
@@ -1,8 +1,10 @@
 const path = require( 'path' );
-const clone = require( '@penskemediacorp/larva' ).clone;// This should be in this repo, probably
-const o_collection_item = clone( path.resolve( __dirname, '../o-collection-item.prototype' ) );
+const clonedeep = require( 'lodash.clonedeep' );
 
-o_collection_item.o_collection_item_classes += " a-floated-parent";
-o_collection_item.o_figure.o_figure_classes += " a-floated-left";
+const o_collection_item_prototype = require( './o-collection-item.prototype' );
+const o_collection_item = clonedeep( o_collection_item_prototype );
+
+o_collection_item.o_collection_item_classes += " lrv-a-floated-parent";
+o_collection_item.o_figure.o_figure_classes += " lrv-a-floated-left";
```

Comment:
should use single quotes to be consistent?

Reply:
Yeah, probably but we don't have any linting on these files at the moment so not too worried, eventually we'll run all of them thru `eslint --fix`  – will update though. Thanks!